app/assets/javascripts/initializePage.js

```diff
@@ -65,6 +65,7 @@ function callInitializers() {
   initializeHeroBannerClose();
   initializeOnboardingTaskCard();
   initializeDateHelpers();
+  initializeColorPicker();
```

Comment:
should the `initializeColorPicker` function exist in the `initializeColorPicker` file?

Reply:
> It works now, thanks!

Thanks Rhymes, not sure how I missed that.